### PR TITLE
Create Product Model

### LINKED ISSUE #2: Create Product Model
Create the model with all attributes in `/backend/models.py`

- `name`* (string)
- `images`* ( [image] )
- `price`* (int)
- `discount` (int)
- `stock` (int)
- `type`* (enum- `jewellery`, `clothes`)
- `description` (string)

Note:
- `*` means required
- All arrays are empty by default
- All non-required numbers are 0 by default

### PR DESCRIPTION
**Pull request by tichnas**
Fixes #2

### CODE REVIEW

**Comment by tichnas** [comment]
Working on images

File: .gitignore [hunk 1]
```diff
@@ -83,6 +83,7 @@ coverage.xml
 local_settings.py
 db.sqlite3
 db.sqlite3-journal
+migrations/
```
Comment (gsc2001) [inline .gitignore]: remove this

File: backend/ecommerce/admin.py [hunk 2]
```diff
@@ -1,3 +1,10 @@
 from django.contrib import admin
+from .models import Product
 
-# Register your models here.
+
+class ProductAdmin(admin.ModelAdmin):
```
Comment (gsc2001) [inline backend/ecommerce/admin.py]: Do you need this ?

File: backend/ecommerce/models.py [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
 
+class Product(models.Model):
+    name = models.CharField(max_length=255)
+    price = models.IntegerField()
+    discount = models.IntegerField(default=0)
+    stock = models.IntegerField(default=0)
 
-class Hii(models.Model):
-    name = models.IntegerField()
+    class Type(models.IntegerChoices):
```
Comment (gsc2001) [inline backend/ecommerce/models.py]: User models.TextChoices
refer [this](https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types)

**Review by gsc2001 — CHANGES_REQUESTED** [review]

**Review by gsc2001 — APPROVED** [review]

**Pull request merged**